chia/wallet/wallet_node.py

```diff
@@ -1016,6 +1045,7 @@ async def validate_received_state_from_peer(
         # Only use the cache if we are talking about states before the fork point. If we are evaluating something
         # in a reorg, we cannot use the cache, since we don't know if it's actually in the new chain after the reorg.
         if await can_use_peer_request_cache(coin_state, peer_request_cache, fork_height):
+            self.log.info("Cache hit!}")
```

Comment:
Maybe a little more explanation too?

```suggestion
            self.log.info("Cache hit!")
```

Reply:
I'll remove this, it was for debugging